src/App.js

```diff
@@ -28,6 +29,7 @@ function App() {
   return (
     <>
       <h1>{message}</h1>
+      <Contact />
```

Comment:
I think you noticed already; but if this is being added, we need to make sure that there's an import at the top of the file too